ansible/todo

```diff
@@ -0,0 +1,15 @@
+todo:
```

Comment:
Ill move some of this to issues since so we can get better visibility into what we'll need to talk over the next couple of weeks